src/compute-client/src/controller.rs

```diff
@@ -292,6 +292,25 @@ impl<T: ComputeControllerTimestamp> ComputeController<T> {
         self.arrangement_exert_proportionality = value;
     }
 
+    /// Returns `true` iff all collections on all clusters have been hydrated.
+    pub fn check_clusters_hydrated(&self) -> bool {
+        let mut result = true;
+        for instance_id in self.instances.keys() {
+            let i = self.instance(*instance_id).expect("known to exist");
+            let instance_hydrated = i.check_any_replica_hydrated();
+
+            if !instance_hydrated {
+                result = false;
+
+                // We continue with our loop, so that we log all non-hydrated
+                // instances.
+                tracing::info!("instance {instance_id} is not hydrated");
```

Comment:
```suggestion
                tracing::info!("cluster {instance_id} is not hydrated");
```

(i think this will be much more understandable in the logs, despite not using the compute controller's favored term)

Reply:
my instinct was to use "cluster", but then I obeyed compute custom... 😅 (will change)